docker-compose.yml

```diff
@@ -120,34 +120,67 @@ services:
     # nodered opens ports on influxdb so it needs to be able to talk to it.
     links:
       - influxdb
+      - postfix
 
   influxdb:
     restart: unless-stopped
-    image: influxdb:1.2.4
+    image: influxdb:latest
```

Comment:
I think we should put the version in the environment variables -- latest is a little tricky in a production environment because we might get unexpected breaking changes. With semantic versions, this would be better as 1.3 or 1.4 or 1 -- so that we don't get breaking changes without human review. However, I'll accept this change for now, as we ought to update.

Reply:
```latest is a little tricky in a production environment because we might get unexpected breaking changes```

yes you are right, I faced issue that i could take influxdb backup previously from extra instance but last week i won't. then i changed  this to latest as above. Then it is working. To avoid such issues we need to maintain same version in the influxdb and influxdb-backup container. 

In the influxdb-backup container, i couldn't install specific version of influxdb package since there is no documentation in their website. I'll try to do this ASAP.

Reply:
```I think we should put the version in the environment variables```
      Ok. I'll do it.